### ISSUE TITLE
Multiple scan conditions on dynamodbcontext c# does not work

### ISSUE THREAD

**Issue by anurita** [aws/aws-sdk-net#922]
I am trying to do something similar to 

DynamoDBOperationConfig config = new DynamoDBOperationConfig()
                {
                    QueryFilter =
                        new List<ScanCondition>()
                        {
                            new ScanCondition("Name", ScanOperator.Equal, "abc"),
                            new ScanCondition("Name", ScanOperator.Equal, "pqr")
                        },
                        ConditionalOperator = ConditionalOperatorValues.Or
                };

                var result = dynamoDBContext.Query<Person>(
                    <hashkey>, config);

Basically, I want to have Conditional OR on the same attribute in dynamo db.
I wanted to know if it is still not supported by dynamo db for c#.net. Also, what can be an alternative solution for this?

[logging]: http://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-config-other.html
